platform/posix/SConscript

```diff
@@ -1,13 +1,11 @@
-import sys
-
 from buildutils import *
 
 Import('env', 'build', 'install')
 localenv = env.Clone()
 
 # Copy man pages
-if env['INSTALL_MANPAGES']:
-    install('$inst_mandir', multi_glob(localenv, '#platform/posix/man', '*'))
+if localenv["OS"] != "Windows" and localenv["INSTALL_MANPAGES"]:
```

Comment:
Is there a need to check against `localenv["OS"]` here? `SConstruct` already does:
```py
if os.name == "nt":
    env["INSTALL_MANPAGES"] = False
```

Reply:
:+1: ... `conda-recipes` deleted the folder, so I assumed it existed.

Reply:
FWIW, the Conda recipe (along with conda-forge) does a lot of things "just in case", or because it's old behavior that hasn't been updated, or just because the files ended up there one time for some reason and the easiest way to fix the problem is `rm -rf` without fixing the underlying cause. So I wouldn't put too much stock in that code as far as interpreting other behaviors are concerned ☺️